src/api/db/models/Camera.ts

```diff
@@ -48,7 +51,7 @@ export class CameraModel {
           const newCamera = new WirelessCamera({
             _id: input.cameraId,
             make: input.make,
-            projRegistrations: [{ projectId, active: true }],
+            projRegistrations: [{ _id: new ObjectId(), projectId, active: true }],
```

Comment:
@nathanielrindlaub another question just for my sake, but what is this id used for here?

Reply:
@jue-henry `ProjectRegistration`s are nested sub-documents in MongoDB parlance, so by default Mongoose was giving them an _id property without us explicitly setting it. To make the Mongoose type inference produce accurate TypeScript typings from our schemas, however, we had to start explicitly setting `_id` fields on all subdocuments (see ProjectRegistration schema [here](https://github.com/tnc-ca-geo/animl-api/blob/7504b98a01c14b0ae459ea01f6328dd2602ca5c4/src/api/db/schemas/WirelessCamera.ts#L4-L8)), which had a knock-on effect of needing to create the `_id`s manually when we create new records instead of letting Mongoose do it under the hood.

You can get more detail on that in this issue: https://github.com/tnc-ca-geo/animl-api/issues/221